### DISCUSSION TITLE
NGINX always 0.0.0.0

### DISCUSSION THREAD

**Discussion by theJuniorProgrammer3** [nginx/nginx#281] [Q&A] [answered]
I haven't used NGINX for a long time. When I wanted to use it again, I changed the configuration in nginx.conf, by changing the "server_name" section from "localhost" to a new domain (which has a different DNS). When I run it, DNS remains 0.0.0.0, unchanged. I have also reinstalled NGINX, and manually re-filled nginx.conf. But the results are the same.
Note: I use Termux, nginx version nginx/1.27.2.
![Screenshot_2024-10-26-13-30-52-408_com.termux.jpg](https://github.com/user-attachments/assets/d7b8b2db-c0de-4442-a283-09d00fce16ed)

Configuration file (after reinstall)

![Screenshot_2024-10-26-13-33-14-148_com.android.chrome.jpg](https://github.com/user-attachments/assets/6e22f5bf-4948-42fa-a3d4-765564e5cba6)

0.0.0.0:8080

**Comment by dvershinin** [comment, marked as answer]
What is the issue? NGINX works. Although having "DNS return 0.0.0.0" is weird, it should point to a real IP address of your server, which should be resolved in your name server configuration anyway, not in NGINX.

**Reply by theJuniorProgrammer3** [reply]
Oh, I haven't used NGINX in a long time so I forget about that. Thanks.
